### DISCUSSION TITLE
LayIM目前版本v3.9.9  layui版本是：layui 2.6.13，layui版本能否我们开发者自行升级？

### DISCUSSION THREAD

**Discussion by hulang** [layui/layui#2184] [问答] [answered]
RT，主要是项目中是layui版本是最新版本：2.9.15。。。怕有冲突。。。

**Comment by sentsim** [comment, marked as answer]
理论上可以直接使用 Layui 2.9.16

**Reply by hulang** [reply]
谢谢作者，我还想问，能否下次更新把：`dropdown`集成到里面去，直接右键好友之类的操场。。
`https://dev.layuion.com/groups/jie/layim/69516/` 这是我很久之前发的帖子，也下次更新到里面。。我每次都是自己查找替换。。
老大，一劳永逸下。。。。

**Reply by sentsim** [reply]
前段时间做了个 AI 有关的项目，顺带把 LayIM 也升级了下，所以近期会发布个 LayIM 4.0，当前还差文档未完成。
你帖子提到的问题已不存在，因为 4.0 是通过函数自主设置聊天记录 URL 的。

**Reply by hulang** [reply]
> 前段时间做了个 AI 有关的项目，顺带把 LayIM 也升级了下，所以近期会发布个 LayIM 4.0，当前还差文档未完成。 你帖子提到的问题已不存在，因为 4.0 是通过函数自主设置聊天记录 URL 的。

期待啊，`tool`扩展工具栏的`icon`是`unicode`格式，更新后的是否也支持`font-class`?
`right`和`minRight`参数，能否更加灵活，支持底部距离等？

**Reply by hulang** [reply]
用户签名，这个参数，能否在聊天窗口用户名称那里显示出来，我都是强制修改到那里的。。。
